src/Microsoft.Health.Fhir.CosmosDb.Initialization/Features/Storage/CollectionInitializer.cs

```diff
@@ -10,60 +10,58 @@
 using Microsoft.Azure.Cosmos;
 using Microsoft.Extensions.Logging;
 using Microsoft.Health.Abstractions.Exceptions;
-using Microsoft.Health.Fhir.CosmosDb.Configs;
-using Microsoft.Health.Fhir.CosmosDb.Features.Storage.Versioning;
+using Microsoft.Health.Fhir.CosmosDb.Core.Configs;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.Versioning;
 using Polly;
 
-namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage
+namespace Microsoft.Health.Fhir.CosmosDb.Initialization.Features.Storage
 {
     public class CollectionInitializer : ICollectionInitializer
     {
         private readonly CosmosCollectionConfiguration _cosmosCollectionConfiguration;
         private readonly CosmosDataStoreConfiguration _cosmosDataStoreConfiguration;
         private readonly IUpgradeManager _upgradeManager;
-        private readonly RetryExceptionPolicyFactory _retryExceptionPolicyFactory;
+
```

Comment:
Probably we can get rid of this empty line.

Reply:
done